adaptive_components/test/adaptive_components_test.dart

```diff
@@ -24,9 +24,9 @@ void main() {
         ),
       );
 
-      var box = tester.widget(find.byType(Container));
+      var box = tester.widget<Container>(find.byType(Container));
```

Comment:
might be able to get away without these `<Container>` annotations

Reply:
See comment above.